app/views/user/check_email.html.slim

```diff
@@ -1,20 +1,23 @@
 .govuk-grid-row
   .govuk-grid-column-two-thirds-from-desktop
     h1.govuk-heading-l Check your email
-
-    p.govuk-body
-      | We have sent you an email with instructions for how to verify your email address.  We sent the email to:
-
-    <p class="govuk-body govuk-!-font-weight-bold">
-      = @user.email
-    </p>
-
-    p.govuk-body
-      | To continue setting up your training account you must click the link in the email. If you have not received the email after a few minutes, please check you spam folder.
+      
+    - if @user
```

Comment:
If there was a comment about popping these in partials I agree

Reply:
Either partials, or two separate templates rendered from the controller, 'check_email_confirmation', 'check_email_password_reset'